modin/core/storage_formats/pandas/parsers.py

```diff
@@ -635,13 +638,31 @@ def parse(fname, **kwargs):
         ]
 
 
+class ParquetFileToRead(NamedTuple):
+    """
+    Class to store path and row group information for parquet reads.
+
+    Parameters
+    ----------
+    path : str, path object or file-like object
+        Name of the file to read.
+    row_group_start : int
+        Row group to start read from.
+    row_group_end : int
+        Row group to stop read.
+    """
+
+    path: Any
+    row_group_start: int
+    row_group_end: int
+
+
 @doc(_doc_pandas_parser_class, data_type="PARQUET data")
 class PandasParquetParser(PandasParser):
     @staticmethod
     @doc(_doc_parse_func, parameters=_doc_parse_parameters_common)
     def parse(fname, **kwargs):
         num_splits = kwargs.pop("num_splits", None)
-        columns = kwargs.get("columns", None)
```

Comment:
why remove, isn't it used later on?

Reply:
I think I should actually get rid of this entire function altogether since it won't be used anymore.